pkg/controller/controllers.go

```diff
@@ -8,19 +8,25 @@ import (
 
 	"github.com/jackc/pgx/v4/pgxpool"
 	policiesv1 "github.com/open-cluster-management/governance-policy-propagator/api/v1"
+	chanv1 "github.com/open-cluster-management/multicloud-operators-channel/pkg/apis/apps/v1"
 	appsv1 "github.com/open-cluster-management/multicloud-operators-placementrule/pkg/apis/apps/v1"
+	subv1 "github.com/open-cluster-management/multicloud-operators-subscription/pkg/apis/apps/v1"
 	configv1 "github.com/stolostron/hub-of-hubs-data-types/apis/config/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	applicationv1 "sigs.k8s.io/application/api/v1beta1"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/scheme"
 )
 
 // AddToScheme adds all the resources to be processed to the Scheme.
-func AddToScheme(s *runtime.Scheme) error {
-	schemeBuilders := []*scheme.Builder{policiesv1.SchemeBuilder, appsv1.SchemeBuilder, configv1.SchemeBuilder}
+func AddToScheme(sch *runtime.Scheme) error {
```

Comment:
what is variable name sch?
this is not readable.

according to best practices, we either give a readable name or a one letter name as it was before this change